package-lock.json

```diff
@@ -387,6 +396,25 @@
       "resolved": "skiplang/prelude/ts/binding",
       "link": true
     },
+    "node_modules/@skipruntime/addon": {
+      "version": "0.0.4",
+      "resolved": "https://registry.npmjs.org/@skipruntime/addon/-/addon-0.0.4.tgz",
+      "integrity": "sha512-YyXRJ9+ZDu0uzYnVfRc/BczlOrFh8akSuwvOKo6yvWYh3rYv04CJKfMjLdZGyaK4D5wh+hJKrNbU2MZDOkGESQ==",
+      "hasInstallScript": true,
+      "optional": true,
+      "dependencies": {
+        "@skipruntime/core": "^0.0.5"
+      }
+    },
+    "node_modules/@skipruntime/addon/node_modules/@skipruntime/core": {
```

Comment:
Are sub-`node_modules` expected?

Reply:
Much (all?) of the package-lock.json stuff here is the result of stale versions in `examples/hackernews/reactive_service/package.json` which are fixed by #691. Should be fine after that lands and this rebases.

The actual docs changes LGTM though!